### ISSUE TITLE
Not using label elements on Mass Controls

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#141]
<img width="581" alt="gflb-pdom" src="https://user-images.githubusercontent.com/1902116/58866838-acf01e00-86b9-11e9-932f-4aef6549ca1c.png">

I just noticed while inspecting the PDOM of GFLB that the `label` for each of the mass controls is actually a `p` element. Ideally, these should be `label` elements - just like for the position sliders.

@zepumph, the implementation does sound ok with VoiceOver, but, in general, it's better to use native HTML (`label` elements with `for` attributes), rather than ARIA (`p`'s and `aria-labelledby`).

@zepumph, perhaps you have taken the ARIA approach because of the custom-nature of these slider-numberspinner object? 

If it is not possible at this point to implement the Mass Controls with `label` elements, we should mark this as an immediate todo for a maintenance release.

@zepumph, could you comment?

**Comment by zepumph** [comment]
@terracoda I made the mass controls have a label for the input. How does that look and sound?

**Comment by terracoda** [comment]
@zepumph, sounds great, looks good in the a11y view, and one improvement needed ;-) ... Now, that you are using native HTML - a `label` element with a `for` attribute - there is not need for the `aria-labelledby` attribute on the `input` element, the `label` and the `input` are associated through the `label`'s `for` attribute. 

Please remove the `aria-labelledby` attribute underlined here in pink, and I think this will do it:
<img width="727" alt="Screen Shot 2019-06-07 at 10 16 22 AM" src="https://user-images.githubusercontent.com/1902116/59091084-d05fd680-890e-11e9-9123-8062b580e008.png">

**Comment by zepumph** [comment]
That makes a lot of sense! How about now?

**Comment by terracoda** [comment]
Thanks @zepumph! Code looks good! And everything sounds good. 
I noticed in the A11y View, that the interactive elements are no longer getting highlights. I am not sure when that happened and it is likely not related to this issue, so I will open a new issue for that, and close this one.
**Screen shot of new label/input code**
<img width="673" alt="mass-label-for" src="https://user-images.githubusercontent.com/1902116/59417969-79ec0f80-8dc8-11e9-8eb3-2e7b47cb9abb.png">
